data/finset.lean

```diff
@@ -992,6 +989,64 @@ sort_eq _ _
 
 @[simp] theorem sort_to_finset [decidable_eq α] (s : finset α) : (sort r s).to_finset = s :=
 list.to_finset_eq (sort_nodup r s) ▸ eq_of_veq (sort_eq r s)
+
 end sort
 
+def disjoint (s t : finset α) : Prop := ∀ ⦃a⦄, a ∈ s → a ∈ t → false
```

Comment:
Just curious why not this?

```lean
def disjoint (s t : finset α) : Prop := s ∩ t = ∅
```

Reply:
Just to fill in some background, I used `s ∩ t = ∅` to allow everything to be simplified to `multiset`'s `disjoint`.

Reply:
I think my form is much easier to use.